### DISCUSSION TITLE
Do we want to use GitHub's discussions?

### DISCUSSION THREAD

**Discussion by markus2330** [ElektraInitiative/libelektra#3586] [Q&A] [answered]
I enabled them for now but if they create inconsistencies with issues, we can also keep having everything in issues.

**Comment by kodebach** [comment, marked as answer]
I think Discussions could be very helpful for people that are just starting to work with and on Elektra. This here is the perfect place for people to ask questions, if they e.g. can't find documentation or they have some very specific problem (something like #3472)

**Comment by kodebach** [comment]
After looking at the discussion in #3606, I think we should also use the Discussions feature when talking about more philosophical questions that don't directly relate to a specific issue or PR.

**Reply by markus2330** [reply]
From a few recent occurrences: I think we should rather stop that. When something might turn out to be an issue or even PR, it should be an issue.
